packages/rpc/src/transports/ipc.ts

```diff
@@ -15,49 +16,59 @@ enum OPCodes {
 	PONG,
 }
 
-function getIPCPath(id: number) {
+export const getIPCPath = (id: number) => {
```

Comment:
Would it be a problem to make this function async in order to use `fs/promises`?